lib/Window.js

```diff
@@ -97,8 +97,13 @@ module.exports = function Window(resp, windowObjects, innerWidth = 760, innerHei
   }
 
   function scroll(xCoord, yCoord) {
-    if (xCoord !== undefined) pageXOffset = xCoord;
-    if (yCoord !== undefined) pageYOffset = yCoord;
+    if (typeof xCoord === "object") {
```

Comment:
One of the quirks of javascript is that the expression `typeof null === object` is true.